### DISCUSSION TITLE
Mongoose doesnt work with next

### DISCUSSION THREAD

**Discussion by ddm4313** [vercel/next.js#15101] [Help]
```
const mongoose = require('mongoose')

mongoose.createConnection('mongodb://localhost:27017/userbase', {
    useNewUrlParser: true,
    useUnifiedTopology: true
});
```
I just get
```
mongoose.createConnection is not a function
mongoose.connect is not a function
``` Mongoose is installed, ive tested many times, it just doesn't work. I ask here because this only happens with Nextjs

**Comment by jamesmosier** [comment]
Have you tried a method similar to the [with-mongodb-mongoose](https://github.com/vercel/next.js/blob/canary/examples/with-mongodb-mongoose) example?

In particular the [dbConnect.js](https://github.com/vercel/next.js/blob/9b711883cfebfce0997cd9fe188a33cd433e9981/examples/with-mongodb-mongoose/utils/dbConnect.js) file.
